server/utils.js

```diff
@@ -13,10 +14,11 @@ const getHistory = () => {
   return JSON.parse(fs.readFileSync(historyPath, "utf8"));
 };
 
-const addCity = (city, temperature) => {
+const addCity = (city, temperature, label = "") => {
   const data = getHistory();
   const date = new Date().toLocaleDateString('fr-FR', { hour: '2-digit', minute: '2-digit' });
-  const el = { city: city, temperature: temperature, date: date};
+  const id = crypto.randomUUID();

```

Comment:
why not uuidv4 ?

Reply:
overkill for this project, simpler to use the native randomUUID from nodejs